rca/events/models.py

```diff
@@ -637,7 +658,7 @@ def event_date_short(self):
 
     @property
     def past(self):
-        return self.end_date.date() < datetime.date.today()
+        return self.end_date < datetime.date.today()
```

Comment:
Would this throw a TypeError if `self.end_date` is None?

Reply:
Yes (and would have before this PR), but `end_date` is a required field

Reply:
Thanks for fixing it